pom.xml

```diff
@@ -828,7 +826,7 @@
                 <goals>
                   <goal>test</goal>
                 </goals>
-                <phase>test</phase>
```

Comment:
For the sake of maintaining Kokoro efficiency, would it be possible to keep the `test` phase? The advantage that it brings over `integration-test` and `verify` is that it doesn't require classes to be packaged/deployed so I'm concerned that using `integration-test` might increase the duration of the builds.

Reply:
Do you mean I should create another `execution`, so we can have two `executions` like
```
<execution>
  <id>test-native</id>
  <goals>
     <goal>test</goal>
  </goals>
  <phase>test</phase>
</execution>
<execution>
  <id>integration-test-native</id>
  <goals>
     <goal>test</goal>
  </goals>
  <phase>integration-test</phase>
</execution>
```